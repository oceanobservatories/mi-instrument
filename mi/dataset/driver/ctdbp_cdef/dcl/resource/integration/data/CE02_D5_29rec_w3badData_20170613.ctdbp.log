2017/06/13 23:30:07.992 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/06/13 23:30:20.889 # 13.2590,  3.65509,    7.043, 13 Jun 2017 23:30:13
2017/06/13 23:30:30.882 # 13.2690,  3.65591,    7.225, 13 Jun 2017 23:30:23
2017/06/13 23:30:40.882 # 13.2674,  3.65472,    6.967, 13 Jun 2017 23:30:33
2017/06/13 23:30:50.881 # 13.2859,  3.65304,    7.338, 13 Jun 2017 23:30:43
2017/06/13 23:31:00.881 # 13.2940,  3.65238,    7.155, 13 Jun 2017 23:30:53
2017/06/13 23:31:10.882 # 13.3099,  3.65186,    7.289, 13 Jun 2017 23:31:03
2017/06/13 23:31:20.881 # 13.3289,  3.65005,    7.390, 13 Jun 2017 23:31:13
2017/06/13 23:31:30.882 # 13.3348,  3.65022,    7.395, 13 Jun 2017 23:31:23
2017/06/13 23:31:40.882 # 13.3436,  3.64980,    7.173, 13 Jun 2017 23:31:33
2017/06/13 23:32:00.950 # 133183,  3.65206,    7.124, 13 Jun 2017 23:31:53
2017/06/13 23:32:10.881 # 13.3036,  3.65446,    7.011, 13 Jun 2017 23:32:03
2017/06/13 23:32:20.882 # 13.2702,  3.65953,    7.138, 13 Jun 2017 23:32:13
2017/06/13 23:32:30.882 # 13.2498,  3.66138,    6.936, 13 Jun 2017 23:32:23
2017/06/13 23:32:40.887 # 13.1906,  3.66811,    7.100, 13 Jun 2017 23:32:33
2017/06/13 23:32:50.882 # 13.1620,  3.66952,    6.964, 13 Jun 2017 23:32:43
2017/06/13 23:33:07.364 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/06/13 23:45:03.059 [ctdbp:DLOGP3]:€SBE 16plus
2017/06/13 23:45:06.479 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/06/13 23:45:19.888 # 13.0570,  3.67001,    7.083, 13 Jun 2017 23:45:12
2017/06/13 23:45:29.882 # 13.0745,  3.66955,    7.196, 13 Jun 2017 23:45:22
2017/06/13 23:45:39.882 # 13.0856,  3.66854,    7.034, 13 Jun 2017 23:45:32
2017/06/13 23:45:49.882 # 13.0589,  3.67029,    7.452, 13 Jun 2017 23:45:42
2017/06/13 23:45:59.881 # 13.0558,  3.67067,    7.195, 13 Jun 2017 23:45:52
2017/06/13 23:46:09.882 # 13.0963,  3.66867,    7.376, 13 Jun 2017 23:46:02
2017/06/13 23:46:19.882 # 13.1159,  3.66784,    7.013, 13 Jun 2017 23:46:12
2017/06/13 23:46:29.893 # 13.1261,  3.66730,    7.123, 13 Jun 2017 23:46:22
2017/06/13 23:46:39.882 # 13.1368,  366683,    6.898, 13 Jun 2017 23:46:32
2017/06/13 23:46:49.882 # 13.1387,  3.66727,    6.913, 13 Jun 2017 23:46:42
2017/06/13 23:46:59.882 # 13.1378,  3.66750,    7.026, 13 Jun 2017 23:46:52
2017/06/13 23:47:09.882 # 13.1433,  3.66750,    6.874, 13 Jun 2017 23:47:02
2017/06/13 23:47:19.881 # 13.1303,  3.66835,    7.589, 13 Jun 2017 23:47:12
2017/06/13 23:47:29.882 # 13.1325,  3.66793,    7.245, 13 Jun 2017 23:47:22
2017/06/13 23:47:39.882 # 13.1259,  3.66831,    7495, 13 Jun 2017 23:47:32
2017/06/13 23:47:49.882 # 13.1245,  3.66846,    7.436, 13 Jun 2017 23:47:42
2017/06/13 23:47:59.882 # 13.1418,  3.66746,    7.487, 13 Jun 2017 23:47:52
2017/06/13 23:48:06.371 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
