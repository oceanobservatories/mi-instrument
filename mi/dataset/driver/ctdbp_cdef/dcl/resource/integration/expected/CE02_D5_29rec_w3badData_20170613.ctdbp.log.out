
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706385420.888999939 |          5 |         3706385413 |  3706385420.888999939 |      port_timestamp |           ok |      3.65509 |             null |                     null |    7.043 |  13.259
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385430.8819999695 |          5 |         3706385423 | 3706385430.8819999695 |      port_timestamp |           ok |      3.65591 |             null |                     null |    7.225 |  13.269
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385440.8819999695 |          5 |         3706385433 | 3706385440.8819999695 |      port_timestamp |           ok |      3.65472 |             null |                     null |    6.967 | 13.2674
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706385450.881000042 |          5 |         3706385443 |  3706385450.881000042 |      port_timestamp |           ok |      3.65304 |             null |                     null |    7.338 | 13.2859
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706385460.881000042 |          5 |         3706385453 |  3706385460.881000042 |      port_timestamp |           ok |      3.65238 |             null |                     null |    7.155 |  13.294
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385470.8819999695 |          5 |         3706385463 | 3706385470.8819999695 |      port_timestamp |           ok |      3.65186 |             null |                     null |    7.289 | 13.3099
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706385480.881000042 |          5 |         3706385473 |  3706385480.881000042 |      port_timestamp |           ok |      3.65005 |             null |                     null |     7.39 | 13.3289
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385490.8819999695 |          5 |         3706385483 | 3706385490.8819999695 |      port_timestamp |           ok |      3.65022 |             null |                     null |    7.395 | 13.3348
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385500.8819999695 |          5 |         3706385493 | 3706385500.8819999695 |      port_timestamp |           ok |       3.6498 |             null |                     null |    7.173 | 13.3436
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706385530.881000042 |          5 |         3706385523 |  3706385530.881000042 |      port_timestamp |           ok |      3.65446 |             null |                     null |    7.011 | 13.3036
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385540.8819999695 |          5 |         3706385533 | 3706385540.8819999695 |      port_timestamp |           ok |      3.65953 |             null |                     null |    7.138 | 13.2702
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385550.8819999695 |          5 |         3706385543 | 3706385550.8819999695 |      port_timestamp |           ok |      3.66138 |             null |                     null |    6.936 | 13.2498
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385560.8870000839 |          5 |         3706385553 | 3706385560.8870000839 |      port_timestamp |           ok |      3.66811 |             null |                     null |      7.1 | 13.1906
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706385570.8819999695 |          5 |         3706385563 | 3706385570.8819999695 |      port_timestamp |           ok |      3.66952 |             null |                     null |    6.964 |  13.162
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386319.8880000114 |          5 |         3706386312 | 3706386319.8880000114 |      port_timestamp |           ok |      3.67001 |             null |                     null |    7.083 |  13.057
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386329.8819999695 |          5 |         3706386322 | 3706386329.8819999695 |      port_timestamp |           ok |      3.66955 |             null |                     null |    7.196 | 13.0745
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386339.8819999695 |          5 |         3706386332 | 3706386339.8819999695 |      port_timestamp |           ok |      3.66854 |             null |                     null |    7.034 | 13.0856
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386349.8819999695 |          5 |         3706386342 | 3706386349.8819999695 |      port_timestamp |           ok |      3.67029 |             null |                     null |    7.452 | 13.0589
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706386359.881000042 |          5 |         3706386352 |  3706386359.881000042 |      port_timestamp |           ok |      3.67067 |             null |                     null |    7.195 | 13.0558
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386369.8819999695 |          5 |         3706386362 | 3706386369.8819999695 |      port_timestamp |           ok |      3.66867 |             null |                     null |    7.376 | 13.0963
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386379.8819999695 |          5 |         3706386372 | 3706386379.8819999695 |      port_timestamp |           ok |      3.66784 |             null |                     null |    7.013 | 13.1159
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706386389.892999649 |          5 |         3706386382 |  3706386389.892999649 |      port_timestamp |           ok |       3.6673 |             null |                     null |    7.123 | 13.1261
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386409.8819999695 |          5 |         3706386402 | 3706386409.8819999695 |      port_timestamp |           ok |      3.66727 |             null |                     null |    6.913 | 13.1387
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386419.8819999695 |          5 |         3706386412 | 3706386419.8819999695 |      port_timestamp |           ok |       3.6675 |             null |                     null |    7.026 | 13.1378
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386429.8819999695 |          5 |         3706386422 | 3706386429.8819999695 |      port_timestamp |           ok |       3.6675 |             null |                     null |    6.874 | 13.1433
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered |  3706386439.881000042 |          5 |         3706386432 |  3706386439.881000042 |      port_timestamp |           ok |      3.66835 |             null |                     null |    7.589 | 13.1303
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386449.8819999695 |          5 |         3706386442 | 3706386449.8819999695 |      port_timestamp |           ok |      3.66793 |             null |                     null |    7.245 | 13.1325
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386469.8819999695 |          5 |         3706386462 | 3706386469.8819999695 |      port_timestamp |           ok |      3.66846 |             null |                     null |    7.436 | 13.1245
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706300800 | telemetered | 3706386479.8819999695 |          5 |         3706386472 | 3706386479.8819999695 |      port_timestamp |           ok |      3.66746 |             null |                     null |    7.487 | 13.1418

(29 rows)
